src/plugins/vis_type_timeseries/common/constants.ts

```diff
@@ -18,3 +18,35 @@
  */
 
 export const MAX_BUCKETS_SETTING = 'metrics:max_buckets';
+
+// Tests Constants
```

Comment:
not sure that we should mix constants for test code and plugin code. I prefer to create a `test_util` folder instead

Reply:
I agree with that, it would be better to have a different file for tests

Reply:
@jasonrhodes should be fixed before merge